Profile parameters

MarekOchocki opened the pull request:
Adding new class for crosswise profile

MarekOchocki committed:
Create crosswise_profile.py

It's empty file

MarekOchocki committed:
new class

grzanka commented on beprof/crosswise_profile.py:
```diff
@@ -0,0 +1,9 @@
+import numpy as np
+
+
+class CrosswiseProfile(profile.Profile):
```
What about simply naming it cross section?

grzanka commented on beprof/crosswise_profile.py:
```diff
@@ -0,0 +1,9 @@
+
+
+class CrosswiseProfile(profile.Profile):
+    def normalize(self):
```
Add a parameter called norm or value with default value 1.0. We may later use it to normalise to 2.0 instead of 1

MarekOchocki committed:
Added new "normalize" method in crosswise_profile.py

MarekOchocki committed:
Added "penumbra" method in crosswise_profile.py

MarekOchocki committed:
Added:
field_ratio
symmetry

and changed:
normalize (to make middle of the profile on x=0)

grzanka commented on beprof/crosswise_profile.py:
```diff
@@ -0,0 +1,45 @@
+
+
+class CrosswiseProfile(profile.Profile):
+    def penumbra(self, reverse=False):
```
Better to introduce left and right penumbras

grzanka commented on beprof/crosswise_profile.py:
```diff
@@ -0,0 +1,45 @@
+        self.y = self.y / ave
+
+    def absolute_y(self):
+        self.y = self.y - self.y.min()
```
Why not using - =?

MarekOchocki committed:
Finish parameters methods in crosswise_profile

grzanka commented:
Will be implemented in https://github.com/grzanka/ibodata
